src/js/views/globals/search/patient-search_views.js

```diff
@@ -90,34 +90,30 @@ const PicklistItem = View.extend({
         {{far "address-card"}}
       </div>
       <div class="patient-search__picklist-item-name-text">
-        <span>{{matchText name search}}{{~ remove_whitespace ~}}</span>
```

Comment:
do we need to remove this?   kinda wonder if we should add this to the match value as well

Reply:
oh or it already is.. are we removing it here because the name shows up as the result?

Reply:
That's an assumption I made. There isn't an example of that scenario in Mark's design.